views/leaguesearch.hbs

```diff
@@ -45,7 +45,27 @@
       </div>
 
     </div>
-  </div>    
+  </div>  
```

Comment:
Can you move this piece into another branch? I agree that it's a good idea, but since we dont have a BE for it. I think we should aim to release without it and push it out in v1.1.

In regards to schema, I would just make a list and then we format it for rendering and push it to a new "leagues" collection. 

Think we should probably get some league logo asssets and what not.
Think w